aws/data_source_aws_iam_policy_document_test.go

```diff
@@ -28,6 +28,34 @@ func TestAccAWSDataSourceIAMPolicyDocument_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSDataSourceIAMPolicyDocument_source(t *testing.T) {
+	// This really ought to be able to be a unit test rather than an
```

Comment:
Agreed! I think this type of refactoring is not necessary for this PR, so not marking that as a requested change  👍

Reply:
That was boilerplate copied from the existing test 😄